checks/remotesettings/uptake_error_rate.py

```diff
@@ -27,36 +32,39 @@ def sort_dict_desc(d, key):
     max_error_percentage: float,
     min_total_events: int = 10000,
     ignore_status: List[str] = [],
+    ignore_versions: List[int] = [],
```

Comment:
Should versions be an `int`? What about 70.1 or 68.0.1?

Reply:
Ok, you're right, that is worth discussing.

I do the `SPLIT(version, ".")[0]` in BigQuery. We could do it here instead. But I supposed that we would ignore major ones only, and this would reduce the amount of data to fetch from the Redash JSON API.

I'm happy to change if you have a strong opinion towards strings, or even regex or whatever

Reply:
No, that's fine, thanks.